src/provider/rainbowKit/RainbowKitAccount.ts

```diff
@@ -212,27 +212,8 @@ class RainbowKitAccount extends Eip6963Provider {
       return Number(parseChainId(CarbonEvmChainIDs[network]))
     }
     const isMainnet = network === Network.MainNet
-    switch (blockchain) {
-      case 'Binance Smart Chain':
-        return isMainnet ? 56 : 97;
-      case 'Mantle':
-        return isMainnet ? 5000 : 5003;
-      case 'Arbitrum':
-        return isMainnet ? 42161 : 421611;
-      case 'Polygon':
-        return isMainnet ? 137 : 80001;
-      case 'OKC':
-        return isMainnet ? 66 : 65;
-      case 'Optimism':
-        return isMainnet ? 10 : 11155420;
-      case 'Avalanche':
-        return isMainnet ? 43114 : 43113;
-      case 'Base':
-        return isMainnet ? 8453 : 84532;
-      default:
-        // Fallback to Ethereum chain ID
-        return isMainnet ? 1 : 5;
-    }
+    // Fallback to Ethereum chain ID
+    return isMainnet ? 1 : 5;
```

Comment:
Think it's best not to hardcode the chain ids here. Instead, maybe you could do something like this:
```typescript
const chainId = isMainnet ? ETH_MAINNET.chainId : ETH_TESTNET.chainId;
return Number(chainId).valueOf(); // convert string hex value into a number
```